### ISSUE TITLE
Doesn't load

### ISSUE THREAD

**Issue by TheAmeliaMay** [CrystalVulpine/Black-Hole#1]
At least on Github Pages: the website never loads; I can't even open the devpane. This is usually caused by never-ending while loops. I suspect the one in main.js on line twenty-two, but I might be wrong.

**Comment by CrystalVulpine** [comment]
No, that loop breaks once it meets all the conditions. It's still working just fine on my end. It even works on my phone. It's probably a browser or graphic specific issue.

**Comment by TheAmeliaMay** [comment]
I checked, and it only works on Firefox-based browsers. I have no idea why, though, since there's no way to actually check what line of code is causing the issue, because it immediately crashes. You'd probably have to code an entirely separate version for Chromium-based browsers just to figure out what the issue is.

**Comment by CrystalVulpine** [comment]
Chrome is my phone browser and it works on that. I also just opened it on Chromium on my computer and it's working fine. If you can't even open the devpane that's definitely some kind of local issue.

**Comment by TheAmeliaMay** [comment]
That's so weird. Try it on Brave, which is the browser I use.
